### ISSUE TITLE
Add install instructions to README.md

### ISSUE THREAD

**Issue by brianezequielfiszman** [polygamma/aurman#250]
## Description
I know it's obvious when you look at the files and requirements for installing **aurman**, but the first thing most people do when entering a repo is reading the documentation and I strongly think that the **Getting Started** and the **Setup** parts are crucial in most of those.

I just installed aurman by cloning and executing **setup.py** and it's not a problem to me, but to some it would be.
Also, the first thing that people do when looking for installing **aurman** is to google it, if it's not on the GitHub documentation, where it would be?

**Comment by Mauladen** [comment]
I think no problem to find aurman in the AUR, download the PKGBUILD and run makepkg
![default](https://user-images.githubusercontent.com/12824589/44239634-020e8200-a1c3-11e8-9a23-b0214c6335e7.PNG)

**Comment by brianezequielfiszman** [comment]
```
wget https://aur.archlinux.org/cgit/aur.git/snapshot/aurman.tar.gz
tar -xvf aurman.tar.gz
cd aurman
makepkg -si
```
@Mauladen If you give me clone permissions I can add it myself, I would have no problem.

**Comment by Morganamilo** [comment]
But why would you use wget over git?

**Comment by eli-schwartz** [comment]
The very first thing in the README, right after the badges (which also point to the AUR):

> **aurman-git** (https://aur.archlinux.org/packages/aurman-git) - **development version**
>
> **aurman** (https://aur.archlinux.org/packages/aurman) - **release version**

**Comment by brianezequielfiszman** [comment]
@Morganamilo because in the AUR page there is the "Download snapshot" link and I used that one instead.

Look, just saying, this AUR Helper is the first choise for those migrating from **yaourt** (including me) and also the first choise for those who are new in **ArchLinux**.
Some of them really know their way around the distro and some are newcomers.

Why not add a Install category? It would help to some people.

```
git clone --recursive https://aur.archlinux.org/aurman-git.git
cd aurman-git
makepkg -si
```

**Comment by Morganamilo** [comment]
The dev had already clearly stated he does not want to do this. Have you read the notice at the top of the readme?

**Comment by brianezequielfiszman** [comment]
@eli-schwartz 
In fact, I'm kinda giving to you the text, it's just some lazy copy-paste, and then

`git checkout develop`
Edit the file
```
git add ./README.md
git commit -m "Added proper Install section"
git checkout master
git merge develop
git push
```

Wow how hard was ir guys?

**Comment by eli-schwartz** [comment]
I'm morally opposed to doing this. Pretty sure polygamma is too.

:)

Also thanks for telling me how to commit a file to git. But it seems rather inefficient to spin off a development branch for such a trivial commit, and also `git commit` can directly commit a file without staging it first, so if you really want to be lazy then:

```
git commit README.md -m "Insert funny commit message"
git push
```

**Comment by brianezequielfiszman** [comment]
@eli-schwartz haven't you checked? The `develop` branch already exists. On the otherside, wow, you know how to use git.

**Comment by polygamma** [comment]
I am impressed by how ignorant this issue is. Even more funny, that you think that @Mauladen is the developer, just because he answered first

**Comment by AladW** [comment]
>To think that complete documentation is unnecessary is elitistic and from what I see, against what the archwiki stands for.

Except the arch wiki takes exactly the same approach as `aurman` in not duplicating information for the sole purpose of handholding ~~help vampires~~ unattentive users...

**Comment by brianezequielfiszman** [comment]
@AladW @polygamma Oh I get it.
Screw newcomers, huh?
And I thought Gentoo was a poisonous and elitist community.

**Comment by brianezequielfiszman** [comment]
@AladW @polygamma Oh I get it.
Screw newcomers, huh?
And I thought **Gentoo** community was a poisonous and elitist.

**Comment by polygamma** [comment]
tl;dr - if you are unable to install aurman without explicit instruction, you are not ready to use an aur helper. that's it.

**Comment by AladW** [comment]
@brianezequielfiszman yep we are totally "poisonous" and "elitist". I'm sure you'll be much happier with the guys over at Manjaro, in the unlikely event you're not already using it.

**Comment by polygamma** [comment]
You are obviously still not ready to use an AUR helper, since you do not grasp the problems which may arise due to the usage of AUR helpers when not having the slightest idea of what they are even doing.

**Comment by eli-schwartz** [comment]
> installing **yaourt** was almost as easy adding a line in pacman.conf and `sudo pacman -S yaourt` (stuff I obviously learnt by reading documentation).

See, this is exactly the kind of person that the Arch Linux community has long refused to support. If you're going to try turning this from an aurman issue into some "the Arch community is elitist and poisonous" argument? Well, it's probably worth noting that supporting such use cases is strictly against the policy of the entire community, indeed.

Also you're insistently and consistently ignoring my initial response.
